docs/server/push-duty.rst

```diff
@@ -157,8 +161,8 @@ someone to work on them.
 Push
 ----
 
-The tag is pushed to production by Jason, once approved by Krupa, on Thursdays.
-It is the responsibility of the push hero to follow-up with Krupa and Jason,
+The tag is pushed to production by ops (wezhou), once approved by QA (Krupa), on Thursdays.
+It is the responsibility of the push hero to follow-up with QA and ops,
```

Comment:
I wonder if we want to call out particular people here, as it's often Adrian instead of Wei; and Krupa hasn't supervised the push for a long, long time - it's the softvision QA folks.